clang/lib/Sema/SemaExprCXX.cpp

```diff
@@ -2008,22 +2008,23 @@ ExprResult Sema::BuildCXXNew(SourceRange Range, bool UseGlobal,
   SourceRange TypeRange = AllocTypeInfo->getTypeLoc().getSourceRange();
   SourceLocation StartLoc = Range.getBegin();
 
-  CXXNewExpr::InitializationStyle initStyle;
+  CXXNewInitializationStyle initStyle;
```

Comment:
```suggestion
  CXXNewInitializationStyle InitStyle;
```
since we're already touching all those lines anyway? (I don't insist.)

Reply:
Done!